test/cls_test.py

```diff
@@ -669,6 +669,26 @@ def bar(self): ...
         assert instance_service_function.warm_pool_size == 5
 
 
+def test_cls_lookup_keep_warm(client, servicer):
+    app = App(name := "my-cls-app")
+
+    @app.cls(serialized=True)
+    class ClsWithMethod:
+        arg: str = modal.parameter(default="")
+
+        @method()
+        def bar(self): ...
+
+    deploy_app(app, name, client=client)
+
+    C = Cls.from_name(name, "ClsWithMethod")
+    obj = C()
+    obj.keep_warm(3)
```

Comment:
we probably already have tests for this, but one thing I've run into in the past when refactoring this code (which is why I think we should have tests for it by now...) is that it's easy to accidentally make stuff re-hydrate on every call. Should make sure that two calls to `.keep_warm()` don't cause a second `FunctionBindParams` request (or `FunctionGet` on the base function) to go out